src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -944,7 +944,8 @@ public override Stream OpenRead(string resolvedPath)
             catch (Exception ex)
             {
                 // line information is only available when PDBs have been emitted
-                var stackTrace = new StackTrace(ex, needFileInfo: true);
+                var needFileInfo = true;
```

Comment:
:-o 

---
In reply to: [130457747](https://github.com/dotnet/roslyn/pull/21046#discussion_r130457747) [](ancestors = 130457747)